sdks/python/apache_beam/internal/auth.py

```diff
@@ -163,4 +164,22 @@ def get_service_credentials():
             user_agent=user_agent)
 
     else:
-      return _GCloudWrapperCredentials(user_agent)
+      try:
+        credentials = _GCloudWrapperCredentials(user_agent)
+        # Check if we are able to get an access token. If not fallback to
+        # application default credentials.
+        credentials.get_access_token()
+        return credentials
+      except AuthenticationException:
+        logging.warning("Unable to find credentials from gcloud.")
```

Comment:
I believe rest of the sdk uses `'` single quotes for log statements, please change them.